config/config.go

```diff
@@ -144,6 +146,74 @@ func (sc *SafeConfig) ReloadConfig(confFile string, logger *slog.Logger) (err er
 	return nil
 }
 
+// CelProgram encapsulates a cel.Program and makes it YAML marshalable.
```

Comment:
Across this PR it would be better to use `CEL` consistently in identifier names, rather than `Cel`, per https://go.dev/wiki/CodeReviewComments#initialisms